Piggy/src/components/homeImage.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import HomePageImage from "../assets/images/home_page_illustration.svg"
+import piggy from './Piggy.png'
+
+export default function homeImage() {
+  return (
+    <div>
+      <p>
+      <img src={piggy} width="800vw" height="550vh"/>
```

Comment:
add alt tag for accessibility. Something like alt="Piggy bank image"

Reply:
No need to add 800vw you can replace by width={800}. You can follow this example: http://blog.matthewcheok.com/a-better-img-tag/